tests/components/mqtt/test_device_tracker_discovery.py

```diff
@@ -440,7 +440,7 @@ async def test_setting_blocked_attribute_via_mqtt_json_message(
         hass,
         mqtt_mock_entry_no_yaml_config,
         device_tracker.DOMAIN,
-        DEFAULT_CONFIG_LEGACY,
+        DEFAULT_CONFIG,
```

Comment:
There's no test using the DEFAULT_CONFIG_LEGACY anymore, should there be one?

Reply:
No, removed it.